crates/net/eth-wire/src/types/broadcast.rs

```diff
@@ -283,25 +288,32 @@ pub struct NewPooledTransactionHashes68 {
     /// instead use the [`Encodable`](alloy_rlp::Encodable) and [`Decodable`](alloy_rlp::Decodable)
     /// implementations for `&[u8]` instead, which encodes into a RLP string, and expects an RLP
     /// string when decoding.
-    #[cfg_attr(
-        any(test, feature = "arbitrary"),
-        proptest(strategy = "proptest::collection::vec(proptest::arbitrary::any::<u8>(), 100)")
-    )]
+  
     pub types: Vec<u8>,
     /// Transaction sizes for new transactions that have appeared on the network.
-    #[cfg_attr(
-        any(test, feature = "arbitrary"),
-        proptest(strategy = "proptest::collection::vec(proptest::arbitrary::any::<usize>(), 100)")
-    )]
+   
     pub sizes: Vec<usize>,
     /// Transaction hashes for new transactions that have appeared on the network.
-    #[cfg_attr(
-        any(test, feature = "arbitrary"),
-        proptest(strategy = "proptest::collection::vec(proptest::arbitrary::any::<B256>(), 100)")
-    )]
+   
     pub hashes: Vec<B256>,
 }
 
+#[cfg(feature = "arbitrary")]
+impl Arbitrary for NewPooledTransactionHashes68 {
+    type Strategy = BoxedStrategy<Self>;
+    type Parameters = ();
+
+    fn arbitrary_with(_args: ()) -> Self::Strategy {
+        (any::<usize>(), any::<u8>(), any::<usize>(), any::<B256>())
+            .prop_map(|(size, type_val, size_val, hashes_val)| NewPooledTransactionHashes68 {
+                types: vec![type_val; size],
+                sizes: vec![size_val; size],
+                hashes: vec![hashes_val; size],
+            })
+            .boxed()
```

Comment:
I'd like to changes this a bit, now that we have this manually

types should be restricted to the TxType values, possible tx types, and ideally randomized vecs